resotocore/resotocore/model/graph_access.py

```diff
@@ -154,6 +161,21 @@ def add_from_json(self, js: Json) -> None:
                 js.get("search", None),
                 js.get("replace", False) is True,
             )
+            if Section.usage in js and len(js[Section.usage]):
+                values = {
+                    k: UsageMetricValues(
+                        min=v.get("min", v["avg"]),
+                        avg=v["avg"],
+                        max=v.get("max", v["avg"]),
+                    )
+                    for k, v in js[Section.usage].items()
+                }
+                usage = UsageDatapoint(
+                    id=js["id"],
+                    at=int(self.at.timestamp()),
```

Comment:
consider creating some variable in the ctor and use it here: `self.at_seconds = int(self.at.timestamp())`